src/main/java/pl/psnc/dei/controllers/requests/UploadDatasetRequest.java

```diff
@@ -0,0 +1,16 @@
+package pl.psnc.dei.controllers.requests;
+
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+import java.util.Set;
+
+@Data
+public class UploadDatasetRequest {
+
+    private @NotNull String datasetId;
```

Comment:
I guess it would be good to include Europeana in the name of the variable to distinguish between datasets from Europeana and Transcribathon

Reply:
renamed variable